mscore/instrwidget.cpp

```diff
@@ -56,6 +56,26 @@ void filterInstruments(QTreeWidget* instrumentList, const QString &searchPhrase)
             for (int cidx = 0; (ci = item->child(cidx)); ++cidx) {
                   // replace the unicode b (accidential) so a search phrase of "bb" would give Bb Trumpet...
                   QString text = ci->text(0).replace(QChar(0x266d), QChar('b'));
+
+                  // replace diacritics with regular latin characters
+                  text = text.replace(QRegularExpression("[\\x{00E1}\\x{00E2}\\x{00E4}\\x{00E0}\\x{00E3}]"), QChar('a'));
+                  text = text.replace(QRegularExpression("[\\x{00E9}\\x{00EA}\\x{00EB}\\x{00E8}]"), QChar('e'));
+                  text = text.replace(QRegularExpression("[\\x{00ED}\\x{00EE}\\x{00EF}\\x{00EC}]"), QChar('i'));
+                  text = text.replace(QRegularExpression("[\\x{00F3}\\x{00F4}\\x{00F6}\\x{00F2}\\x{00F5}]"), QChar('o'));
+                  text = text.replace(QRegularExpression("[\\x{00FA}\\x{00FB}\\x{00FC}\\x{00F9}]"), QChar('u'));
+                  text = text.replace(QRegularExpression("[\\x{00E7}]"), QChar('c'));
+                  text = text.replace(QRegularExpression("[\\x{00F1}]"), QChar('n'));
+                  text = text.replace(QRegularExpression("[\\x{00FD}]"), QChar('y'));
+
+                  text = text.replace(QRegularExpression("[\\x{00C1}\\x{00C2}\\x{00C4}\\x{00C0}\\x{00C3}]"), QChar('a'));
+                  text = text.replace(QRegularExpression("[\\x{00C9}\\x{00CA}\\x{00CB}\\x{00C8}]"), QChar('e'));
+                  text = text.replace(QRegularExpression("[\\x{00CD}\\x{00CE}\\x{00CF}\\x{00CC}]"), QChar('i'));
+                  text = text.replace(QRegularExpression("[\\x{00D3}\\x{00D4}\\x{00D6}\\x{00D2}\\x{00D5}]"), QChar('o'));
+                  text = text.replace(QRegularExpression("[\\x{00DA}\\x{00DB}\\x{00DC}\\x{00D9}]"), QChar('u'));
+                  text = text.replace(QRegularExpression("[\\x{00C7}]"), QChar('c'));
+                  text = text.replace(QRegularExpression("[\\x{00D1}]"), QChar('n'));
+                  text = text.replace(QRegularExpression("[\\x{00DD}]"), QChar('y'));
+
                   bool isMatch = text.contains(searchPhrase, Qt::CaseInsensitive);
```

Comment:
Instead you could directly use a `contains()` variant that works with regular expressions, like `bool QString::contains(const QRegularExpression &re) const`.
Also what about e.g. the Dutch `ij` and the German `ß`. Which characters are you covering, can you add a comment about that, e.g. mentioning their HTML entity, like `&auml;` for `ä`?